pixi.toml

```diff
@@ -0,0 +1,67 @@
+# Pixi is a package management tool for developers.
+# Before running a task, pixi ensures that all listed dependencies are installed first.
+#
+# Run the tasks with `pixi run TASK`, e.g. `pixi run codegen`.
+# Extra CLI argument after `pixi run TASK` are passed to the task cmd.
+# Lists tasks with `pixi task list`
+#
+# Type `pixi shell` to enter the pixi environment.
+#
+# https://prefix.dev/docs/pixi/overview
+
+[project]
+name = "rerun"
+authors = ["Jeremy Leibs <jeremy@rerun.io>"]
+channels = ["conda-forge"]
+description = "Log images, point clouds, etc, and visualize them effortlessly"
+homepage = "https://rerun.io"
+license = "MIT OR Apache-2.0"
+platforms = ["linux-64", "osx-arm64", "osx-64", "win-64"]
+readme = "README.md"
+repository = "https://github.com/rerun-io/rerun"
+version = "0.1.0"                                                              # TODO(emilk): sync version with `Cargo.toml` with help from `crates.py`
```

Comment:
todo is fine, but we might as well just start with `0.10.0-alpha`

Reply:
I think this is more clearly wrong. 0.10.0-alpha will just bitrot quickly. We're not planning on publishing rerun on pixi soon.